docs/api-testing/contract-testing.md

```diff
@@ -0,0 +1,82 @@
+
+If this contract is broken by either party, it can lead to bugs and malfunctions. _API Contract Testing_ is the act of validating that the API producer and the API consumer are respecting the contract. That’s where Sauce Labs API Testing comes in.
+
+## What You'll Need
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+
+## Use Cases
+
+Contract testing is a fast, lightweight form of API testing that strictly checks the content and format of requests and responses. This method is ideal for:
+* Testing APIs during the early stages of design and development
+* Organizations creating APIs that are internal and/or have limited number of consumers
+
+This is typically done in a protected, static environment, where tests are run against mocked (not live) APIs, allowing contract tests to compare isolated API responses to the contract for immediate attention if something is wrong.
+
+<img src={useBaseUrl('img/api-fortress/2022/03/api-consumer-contract.png')} alt="API Conversation and Contract" width="600"/>
+
+## Testing the API Producer Side
+To test the producer (server) side:
+1. From an API Testing Project, go to the HTTP Client.
+2. Import an OpenAPI specification file (v3.0 or higher).
+3. From your list of **Snapshots**, choose the API call you'd like to test by clicking on it. The HTTP method, request URL, and anything else specified in the spec file will populate in the HTTP Client fields.
+4. Click **Send** to send your request.
+5. Click **Generate Test**.
+
+Sauce Labs API Testing will validate the API producer side by creating a contract tests from your OpenAPI spec file.
+<img src={useBaseUrl('img/api-fortress/2022/03/api-producer-contract.png')} alt="API Conversation and Contract" width="500"/>
+
+After you generate your test, you'll be taken to the **Compose** tool. Optionally, you can add further [assertions](/api-testing/composer/) to your test, which will perform functional end-to-end testing on top of your contract tests and fully validate the APIs are working as intended.
+
+<details><summary>What is functional testing?</summary>
+<i>Functional (end-to-end) testing</i> is a more robust, data-driven method that checks the API logic and consumer flows. If your organization is creating a large-scale API program that will have public APIs with third-party consumers, for example, functional testing is ideal. That's where adding functional testing to complement your contract testing strategy can give your development team insight into how accurately your APIs render, and ultimately bring products to market faster.
+</details>
+
+You can view your contract test's results and events on your [Project Dashboard](/api-testing/project-dashboard/).
```

Comment:
@kimsauce please add a note that user can see logs only if test has been executed outside the Composer (following the steps above user is on the Composer)

Reply:
To confirm, outside the Composer means using CLI (Piestry)?

Reply:
You wrote this: _"After you generate your test, you'll be taken to the **Compose** tool. [...]"_ then this _"You can view your contract test results and events on your [Project Dashboard](/api-testing/project-dashboard/)."_
To me, this means you suppose user runs the test from within the Composer, but running a test inside the Composer does not generate any log in the dashboard. This is not related to "Contract testing" but a general rule for all the test you create.

Piestry is a mocking server does not execute tests. To generate a log you should run the test from Test List (other possibilities are: using our APIs or the CLI apifctl but I don't think user generate a test inside the platform and then leave it for running the test using another tool)